business_requirement/models/business.py

```diff
@@ -0,0 +1,446 @@
+        [
+            ('normal', 'In Progress'),
+            ('on_hold', 'On Hold'),
+            ('done', 'Ready for next stage')
+        ],
+        'Kanban State',
+        track_visibility='onchange',
+        required=False,
+        default='normal'
+    )
+    origin = fields.Char(
+        string='Source',
+        readonly=True,
+        states={
+            'draft': [('readonly', False)],
+            'confirmed': [('readonly', True)]
+        }
+    )
+
+    @api.multi
+    @api.onchange('project_id')
+    def project_id_change(self):
+        if self.project_id and self.project_id.partner_id:
+            self.partner_id = self.project_id.partner_id.id
+
+    @api.model
+    def create(self, vals):
+        if vals.get('name', '/') == '/':
+            vals['name'] = self.env['ir.sequence'].next_by_code(
+                'business.requirement')
+        if vals.get('project_id'):
+            project_id = self.env['project.project']. \
+                browse(vals.get('project_id'))
+            if project_id and project_id.message_follower_ids:
+                msg_followers = []
+                for project in project_id.message_follower_ids:
+                    msg_vals = {
+                        'channel_id': project.channel_id.id,
+                        'display_name': project.display_name,
+                        'partner_id': project.partner_id.id,
+                        'res_model': self._name,
+                        'subtype_ids': project.subtype_ids.ids
+                    }
+                    msg_followers.append((0, 0, msg_vals))
+                if msg_followers:
+                    vals['message_follower_ids'] = msg_followers
+        return super(BusinessRequirement, self).create(vals)
+
+    @api.multi
+    def write(self, vals):
+        if vals.get('project_id'):
+            project_id = self.env['project.project']. \
+                browse(vals.get('project_id'))
+            if project_id and project_id.message_follower_ids:
+                msg_followers = []
+                for followers in self.message_follower_ids:
+                    msg_followers.append((2, followers.id))
+                for project in project_id.message_follower_ids:
+                    msg_vals = {
+                        'channel_id': project.channel_id.id,
+                        'display_name': project.display_name,
+                        'partner_id': project.partner_id.id,
+                        'res_model': self._name,
+                        'subtype_ids': project.subtype_ids.ids
+                    }
+                    msg_followers.append((0, 0, msg_vals))
+                if msg_followers:
+                    vals['message_follower_ids'] = msg_followers
+        if vals.get('state'):
+            ir_obj = self.env['ir.model.data']
+            br_xml_id = ir_obj.\
+                get_object('business_requirement',
+                           'group_business_requirement_manager')
```

Comment:
You can use self.env.ref('business_requirement.group_business_requirement_manager')